translations/LICENSE_zh-CN

```diff
@@ -1,11 +1,13 @@
 GLWT（祝你好运）公共许可证
-版权所有（C）每个人，除了作者
+版权所有（C）每个人，包括作者
```

Comment:
This translates to "including the author" instead of "except the author". This line should be undone.